src/include/OpenImageIO/fmath.h

```diff
@@ -897,7 +897,7 @@ inline T safe_pow (T x, T y) {
     // less than FLOAT_MAX (but still bigger than anyone is likely to
     // need) so that subsequent additions or multiplications are less
     // likely to overflow and end up with an Inf right afterwards.
-    const T big = T(1e30);
+    const T big = std::numeric_limits<T>::max();
```

Comment:
I agree. `pow` with large arguments is probably not that common. We do our best to safeguard the common pitfalls where its cheap to do so, but there's no particularly good default for this one.

Nuke the comment and it LGTM.